input/otlp/metrics_test.go

```diff
@@ -608,149 +576,44 @@ func TestConsumeMetrics_JVM(t *testing.T) {
 	metricSlice := scopeMetrics.Metrics()
 
 	timestamp := time.Unix(123, 0).UTC()
-	addInt64Sum := func(name string, value int64, attributes map[string]interface{}) {
+
+	addInt64Gauge := func(name string, value int64, attributes map[string]interface{}) {
 		metric := metricSlice.AppendEmpty()
 		metric.SetName(name)
-		sum := metric.SetEmptySum()
-		dp := sum.DataPoints().AppendEmpty()
+		gauge := metric.SetEmptyGauge()
+		dp := gauge.DataPoints().AppendEmpty()
 		dp.SetTimestamp(pcommon.NewTimestampFromTime(timestamp))
 		dp.SetIntValue(value)
 		dp.Attributes().FromRaw(attributes)
 	}
-	addInt64Gauge := func(name string, value int64, attributes map[string]interface{}) {
+	addInt64Histogram := func(name string, counts []uint64, values []float64, attributes map[string]interface{}) {
 		metric := metricSlice.AppendEmpty()
 		metric.SetName(name)
-		gauge := metric.SetEmptyGauge()
-		dp := gauge.DataPoints().AppendEmpty()
+		histogram := metric.SetEmptyHistogram()
+		dp := histogram.DataPoints().AppendEmpty()
 		dp.SetTimestamp(pcommon.NewTimestampFromTime(timestamp))
-		dp.SetIntValue(value)
+		dp.BucketCounts().Append(1, 1, 2, 3)
+		dp.ExplicitBounds().Append(0.5, 1.5, 2.5)
```

Comment:
[for reviewer] I've just used values that allow the test to pass as I am not familiar with how histograms are stored or could be transformed. The test here is more about "getting metrics stored as-provided" rather than realistic values.